src/config/constants.js

```diff
@@ -1,2 +1,5 @@
 export const TOKEN_URL = 'http://localhost:8080/token';
 export const MAIN_URL = 'https://pfa.foreca.com/api/v1';
+export const numberZero = 0;
+export const numberOne = 1;
+export const numberThree = 3;
```

Comment:
Please use more informative names according to business logic. Zero is not required to be defined as variable :)

Reply:
The hardest part is always to give names for variables... :) but I fixed to `minSearchCharacters = 3`

p.s. you saved me for give name to number 1.